R/code_agreements.R

```diff
@@ -8,11 +8,15 @@
 #' the function finds title and date conforming columns
 #' in the dataset automatically if available.
 #' @param title title column variable.
-#' Ideally, title variable should come from a qPackage database/dataset
-#' for which strings were standardised with `standardise_titles()`
+#' Ideally, title variable should come
```

Comment:
The function "expects" that there is a variable named `Title` and that it has been standardised using `standardise_titles()`